ToDoApp/containers/newTask/NewTask.js

```diff
@@ -37,9 +40,15 @@ export class NewTask extends Component {
     ),
   });
 
-  addTask(navigation){
-    navigation.push('Home');
+  firstInputOnChange(text){
```

Comment:
Please use idiomatic names for functions and variables, in this case probably a good name would be `onChangeName`

Reply:
use arrow functions when possible too